lib/hammer_cli_foreman_azure_rm/compute_resources/azure_rm.rb

```diff
@@ -30,10 +30,11 @@ def host_attributes
 
       def interface_attributes
         [
-          ['public_ip',  _('Public IP (None, Static, Dynamic)')],
-          ['network',    _('Select one of available Azure Subnets, must be an ID')],
-          ['private_ip', _('Static Private IP (expressed as 0 or 1)')]
+          ['compute_network',    _('Select one of available Azure Subnets, must be an ID')],
```

Comment:
@shiramax Also, adding the `compute_` prefix, doesn't work for me in case of creating host using compute profile. Any thoughts?